features/collection/persistence-rrd/src/main/java/org/opennms/netmgt/collection/persistence/rrd/RrdPersisterFactory.java

```diff
@@ -87,5 +102,17 @@ public ResourceStorageDao getResourceStorageDao() {
     public void setResourceStorageDao(ResourceStorageDao resourceStorageDao) {
         m_resourceStorageDao = resourceStorageDao;
     }
+
+    private Persister loadKafkaPersister(ServiceParameters params, RrdRepository repository) {
+        Boolean enabled = new Boolean(System.getProperty("org.opennms.timeseries.kafka.persister", "false"));
+        Persister persister = null;
+        if (enabled) {
+            final ServiceLookup serviceLookup = new ServiceLookupBuilder(DefaultServiceRegistry.INSTANCE).blocking()
+                    .build();
+            PersisterFactory persisterFactory = serviceLookup.lookup(PersisterFactory.class, "(strategy=kafka)");
```

Comment:
I'd prefer to not have to extend the existing persisters with knowledge of another specific persister.

Reply:
Can this be removed?